route.php

```diff
@@ -132,6 +132,13 @@ public function decode($q, $documentRoot, $requestMethod)
         } elseif(in_array($requestMethod, array('POST', 'DELETE', 'PUT'))) {
             $this->method = $requestMethod;
         }
+        if($requestMethod === 'OPTIONS') {
```

Comment:
```suggestion
        elseif($requestMethod === 'OPTIONS') {
```

Reply:
added html to my edits of a php file!!
```
span class="pl-s1"> 
```

use this new github feature with caution! (still beta)